config/read.go

```diff
@@ -364,6 +364,13 @@ func preprocessConfig(config *ServerConfig) (*ServerConfig, error) {
 				config.CrunchyBridgeClusterID = parts[1]
 			}
 		}
+	} else if strings.HasSuffix(host, ".aivencloud.com") {
+		parts := strings.SplitN(host, ".", 2)
```

Comment:
It appears that the Aiven format is actually `serviceId-projectId.aivencloud.com`.

For example, with a project named `myexample12345` and a service named `pg-1234abcd`, the hostname thats generated is `pg-1234abcd-myexample12345.aivencloud.com`.

I think its consistent that aiven uses a single dash in the service ID (and you can't modify that), so I wonder if we should split that here? (everything after the second dash is the project ID, everything before is the service ID)

If we were to call the API in the future, that also requires this split: https://api.aiven.io/doc/#operation/PGServiceQueryStatistics

Reply:
Yeah, I noticed that but figured the identifier was opaque and didn't want to risk a format change from Aiven breaking things on our end. But future API integration is a good point. Should we track the project name separately then? Or can we ignore it?

Reply:
I think tracking it separately seems like the right way to go.

Reply:
Done that way in 2f21961a40467d4d1837bdd5f2d34341f142756f